Automattic/jetpack
# CLI: Add version to Jetpack Release

#### Changes proposed in this Pull Request:

Adds `jetpack release [project] version` to the CLI, which lets you bump the version in composer.json, package.json and the main plugin's php file. It also auto suggests versions for you.

#### Jetpack product discussion


N/a

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






Try using `jetpack release` and following the prompts. 
Try passthroughs, such as `jetpack release plugins/jetpack version`
Try using some flags:
-a (alpha)
-b (beta)
-s (stable)
e.g. `jetpack release plugins/jetpack version -a` should prompt the next alpha release.

`plugins/jetpack` alpha releases should be in format x.y-a.z, all other should be x.y-alpha.
